aetherpy/tests/test_utils_time.py

```diff
@@ -98,20 +98,29 @@ def update_lon(self, multi_lon):
             self.lon = np.full(shape=len(self.times), fill_value=self.lon)
         return
 
+    @pytest.mark.parametrize("lon_sign", [1, -1])
     @pytest.mark.parametrize("multi_lon", [True, False])
-    def test_ut_to_lt(self, multi_lon):
+    def test_ut_to_lt(self, lon_sign, multi_lon):
```

Comment:
Does this work for non-Earth planets? Should we add in tests for this?

Reply:
Good question!  I do not think the routine would work for other planets because it uses datetime input, and so is converting from universal time to solar local time (using two earth-based assumptions: location of prime meridian and hours/degrees longitude).  Introducing other planets would be outside the scope of this pull request, but be a good thing to do.  Would you like to create an issue in the repository for this?